build/setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+export AWS_DEFAULT_REGION=us-east-2
+export AWS_ACCESS_KEY_ID=test
+export AWS_SECRET_ACCESS_KEY=test
+export ENDPOINT=http://localstack:4566
+
+timeout=120 #seconds
+while [ "$(curl -s $ENDPOINT/health | jq '.services.s3')" != "\"running\"" ]; do
```

Comment:
Maybe we can integrate this healtcheck inside the CI: [link](https://github.com/localstack/localstack/issues/1095#issuecomment-505368702)